### PR TITLE
feat: add KeyStatistics component for next theme

### PR DESCRIPTION
**Pull request by dcshzj**
Figma: https://www.figma.com/file/8TEzSSDbx97gctRJ7nKdqG/Website-Template-Gen-2?type=design&node-id=1169-9441&mode=dev

### CODE REVIEW

**Comment by dcshzj** [comment]
!run chromatic

**Review by jacobkwan — COMMENTED** [review]
responsive behaviour looks good

some minor observations:
1. padding x and y is off for mobile, tablet and desktop (not enough)
2. font size for `title` and `value` should be smaller on mobile
3. text color should be different for `title`, `label` and `value`
4. font weight should be 600 for `title` and `value`

for 3. I'll be adding it to tailwindconfig so can ignore for now; I'll fix all the hardcoded colors at one go when I do it

**Comment by dcshzj** [comment]
@jacobkwan fixed all in fe8f6a57e784888c5faefd9cb116201dbaca6b7b. The colour I hardcoded for now.

File: src/templates/next/components/KeyStatistics/KeyStatistics.tsx [hunk 1]
```diff
@@ -19,7 +19,9 @@ const KeyStatistics = ({ variant, title, statistics }: KeyStatisticsProps) => {
           .slice(0, variant === "side" ? 3 : 4)
           .map(({ label, value }) => (
             <div className="flex flex-col gap-3">
-              <h3 className="text-5xl leading-[3.5rem]">{value}</h3>
+              <h3 className="text-5xl leading-[3.5rem] text-[#2c2e34] font-semibold">
```
Comment (jacobkwan) [inline src/templates/next/components/KeyStatistics/KeyStatistics.tsx:22]: mobile font size for `value` should be same as `title` too!
feel free to merge when that's in

**Review by jacobkwan — APPROVED** [review]

File: src/templates/next/components/KeyStatistics/KeyStatistics.tsx (hunk 1 above)
Comment (dcshzj) [reply src/templates/next/components/KeyStatistics/KeyStatistics.tsx]: Oops lol I missed out this one. Adjusted in 4533d9f168bbf9b97cef91e07788d24d50ee13d0.

**Pull request merged**